Samples/MNIST/UWP/cs/DXCore_WinRTComponent/DXCoreHelper.idl

```diff
@@ -0,0 +1,12 @@
+namespace DXCore_WinRTComponent
+{
+    [default_interface]
+    runtimeclass DXCoreHelper
+    {
+        DXCoreHelper();
+        Windows.AI.MachineLearning.LearningModelDevice GetDeviceFromVpuAdapter();
+        Windows.AI.MachineLearning.LearningModelDevice GetDeviceFromComputeOnlyAdapter();
+        Windows.AI.MachineLearning.LearningModelDevice GetDeviceFromGraphicsAdapter();
+        Windows.Foundation.Collections.IVectorView<Windows.AI.MachineLearning.LearningModelDevice> GetAllHardwareDevices();
```

Comment:
these are not referenced in your sample, do you want to keep them around to show their implementation? is there a readme.md or something else that would reference them? If they are not required, I would suggest you declare only a static method in this WinRT Component and let go of the constructor.

Reply:
Fixed - this code is copied from the standalone DXCore_WinRTComponent sample which has its own readme. I will port over your feedback into the the main copy of the sample separately.